HISTORY.rst

```diff
@@ -3,6 +3,140 @@
+* docker-compose.yml: Make it consistent with Dockerfile #1197 
+* Improve Dockerfile #1193 
+* update to include Matrix and GitHub discussion links #1179 
+* Update docs to remove old discussion forums #1171 
+* Add "Grass" curve to wolf_sheep example chart #1178 
+* feat: Implement random activation by type #1162 
+
+* mypy 
+    * mypy: Use "|" operator instead of Union/Optional #1345
+    * mypy: Improve space.py annotation, part 2 #1219 
+    * mypy: Improve annotations #1212 
+
+* Userparam class updates
+    * feat: Implement NumberInput UserParam class #1343 
+    * feat: Implement StaticText UserParam #1342 
+    * feat: Implement Choice UserParam class #1338 
+    * feat: Implement Checkbox UserParam class #1332 
+    * feat: Implement Slider UserParam class #1272
+        * examples: Convert to using Slider UserParam class #1340 
+
+* Front-end updates
+    * frontend: Add alignment options to agent portrayals in CanvasGridVisualization #1349
+    * frontend: Update Bootstrap 4.6.1 -> 5.1.3 #1325
+    * ChartModule.js: Use more semantic HTML element creation method #1319 
+    * Issue #1232; Replaced usage of var to const/let in some files #1248 
+    * [Issue 1232] Refactor NetworkModuleSigma PieChartModule TextModule JS #1246
+    * js: Update D3 from 4.13.0 to 7.4.3 #1270 
+    * support package and local css includes #1283 
+    * Upgrade to Bootstrap 4! #1282
+    * refactor: update var to const/let in InteractionHandler.js #1273 
+    * Change remaining vendored JS dependencies to be downloaded during install #1268 
+    * Download jQuery and D3 during install #1260
+    * CSS support for custom visualization element #1267 
+    * style: prettify js files #1266 
+    * refactor: Change var to const/let for remaining js files #1265 
+    * Remove NetworkModule_sigma and its dependencies #1262 
+    * js: Download bootstrap-slider during install #1257 
+    * js deps: Move Bootstrap to be inside external folder #1236 
+    * Apply prettier to NetworkModule_d3.js #1225 
+    * js: Download Bootstrap on-the-fly during install instead #1220 
+    * Install JS dependencies using Fanstatic #1195 
+    * JQuery updates
+        * examples: Remove all usage of jQuery #1356
+        * Remove jQuery dependency completely #1355
+        * refactor: frontend: Remove remaining usage of jQuery #1351 
+        * refactor: frontend: Remove usage of jQuery for most of the JS code #1348 
+        * refactor: frontend: Remove jQuery usage in CanvasHexModule.js & CanvasModule.js #1347 
+        * refactor: frontend: Remove jQuery usage in BarChartModule.js #1326 
+        * visualization: Specify tooltip without jQuery #1308
+
+* Implement simplified namespace
+    * docs: simplified namespace tutorial update #1361 
+    * examples: Convert shape_example, sugarscape_cg, virus_on_network, wolf_sheep to simple namespace #1339 
+    * Convert hex_snowflake, pd_grid, schelling to simple namespace; [BREAKING] Remove class name collision #1333 
```

Comment:
This PR contains a commit breaking change: "BREAKING: Rename mesa.visualizations.TextVisualization.TextElement to ASCIIElement" that needs to be highlighted.

Reply:
should that be highlight at the top or in place?

Reply:
I think there should be a section with breaking changes at the top. The js dependency updates are also breaking changes, since they are user facing updates.